applications/KubeArmor/Chart.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
```

Comment:
# SPDX-License-Identifier: Apache-2.0
# Copyright (c) 2022 Accuknox

Can you please add this license header on all the yaml files as well as README.md. Basically every file which is getting uploaded here needs to have that.